site/content/en/docs/user/examples/injectfailure/init-container-error-injection.demo

```diff
@@ -0,0 +1,36 @@
+# Demo: inject fault to initContainer in an existing minikube cluster
+kubectl config use-context minikube
+
+kubectl get nodes
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+WORK_DIR=$(cd ${SCRIPT_DIR}/../site/content/en/docs/user/examples/failureinjection/; pwd -P)
```

Comment:
#1066
I think it should be a builtin variable

Reply:
OK, I'll make a change after your PR is merged.